packages/@aws-cdk/aws-lambda-event-sources/README.md

```diff
@@ -116,21 +116,34 @@ To process events with a Lambda function, first create or update a DynamoDB tabl
 and add it to your Lambda function. The following parameters will impact Amazon DynamoDB's polling behavior:
 
 * __batchSize__: Determines how many records are buffered before invoking your lambda function - could impact your function's memory usage (if too high) and ability to keep up with incoming data velocity (if too low).
+* __bisectBatchOnError__: If a batch encounters an error, this will cause the batch to be split in two and have each new smaller batch retried, allowing the records in error to be isolated.
+* __maxBatchingWindow__: The maximum amount of time to gather records before invoking the lambda. This increases the likelihood of a full batch at the cost of possibly delaying processing.
+* __maxRecordAge__: The maximum age of a record that will be sent to the function for processing.
```

Comment:
Add a short phrase or sentence about what happens to records that exceed this age.